### DISCUSSION TITLE
[Listbox] Option with <Link /> inside does not close the Listbox when clicked

### DISCUSSION THREAD

**Discussion by acavestro** [tailwindlabs/headlessui#2954] [Help]
Hello,
consider this codepen: https://codepen.io/acavestro/pen/zYbRyNd?editors=0011.

If there is a `<Link />` inside the `<Listbox.Option />` it seems that the Listbox does not close anymore when an option is selected. I can't understand how to avoid this without having to somehow manually handle the open/close mechanism. 

The `<Link />` component comes from `react-router-dom` package inside `remix`.

Has it ever happened to you?

Thanks 🙏 

PS I'm working with headlessui version 1.7.17, remix v2.1.0 which uses react-router-dom 6.17.0

**Comment by kevinberlanga** [comment]
This is currently happening to me! Were you able to find a fix? I'm using `<Link />` from next/link

**Comment by ThijmenGThN** [comment]
Use ListboxButton instead, example:

```tsx
<ListboxButton as={Link} href="/dash/projects">
  Manage projects
</ListboxButton>
```
